chat-commands.js

```diff
@@ -2479,6 +2479,59 @@ exports.commands = {
 	},
 	blacklisthelp: [`/blacklist [username], [reason] - Blacklists the user from the room you are in for a year. Requires: # & ~`],
 
+	bb: 'battleban',
+	battleban: function (target, room, user) {
+		if (!target) return this.parse('/help battleban');
+
+		let reason = this.splitTarget(target, true);
+		let targetUser = this.targetUser;
+
+		if (!targetUser) {
+			return this.errorReply(`User '${this.targetUsername}' not found.`);
+		}
+		if (!this.can('forcerename', targetUser)) return false;
```

Comment:
Why not `lock` assuming this is going to be for %+ and not @+ (iirc the policy disscusion never finished)

Reply:
I copied most of this code from a different command, yeah I can change it to lock

Reply:
Forcerename is available to drivers so this is %+. Yeah the policy wasn’t finalized, I just did it like this since it’s the one I most agreed with. Gdrivers deserve battle moderation as well as server moderation imo. It seems strange to restrict global moderation from global staff.

Reply:
Or, how about we don't overlap permissions and just create new ones for new features, that works too

Reply:
Why? is this preferred now a days?